hkpy/hklib/entity.py

```diff
@@ -84,12 +84,35 @@ def to_dict(self) -> Dict:
                 jobj['type'] = v.value
             elif k == 'class_name':
                 jobj['className'] = v
+            elif k == 'properties':
+                jobj['properties'] = v
             elif k == 'metaproperties':
                 jobj['metaProperties'] = v
-            else:
```

Comment:
There is a huge problem in removing this else. All HKEntity attributes that are no specified in the above if-elif are now ignored. For example: `interfaces`, `binds`, among others.